php/Blocks/Loader.php

```diff
@@ -473,19 +486,25 @@ protected function block_template( $name, $type = 'block' ) {
 
 			// This is not a load once template, so require_once is false.
 			load_template( $theme_template, false );
-		} else {
-			if ( ! current_user_can( 'edit_posts' ) || ! isset( $templates[0] ) ) {
-				return;
-			}
-			// Hide the template not found notice on the frontend, unless WP_DEBUG is enabled.
-			if ( ! is_admin() && ! ( defined( 'WP_DEBUG' ) && WP_DEBUG ) ) {
-				return;
-			}
+			return;
+		}
+
+		if ( ! empty( $this->blocks[ "genesis-custom-blocks/{$name}" ]['templateMarkup'] ) ) {
+			$this->template_editor->render_markup( $this->blocks[ "genesis-custom-blocks/{$name}" ]['templateMarkup'] );
+			return;
+		}
+
+		if ( ! current_user_can( 'edit_posts' ) || ! isset( $templates[0] ) ) {
```

Comment:
I wonder if it makes sense to introduce a custom user permission for this? `Maybe current_user_can ( 'edit_genesis_custom_blocks' )` or something?

Reply:
Hm, we could use [genesis_custom_block_edit_block](https://github.com/studiopress/genesis-custom-blocks/blob/1ba74a3f25908f49b28b32b26a26396a747b7153/tests/php/Integration/TestPostCapabilities.php#L101), but I'm not too sure it would make a big difference.

Here's the main context they would see this in:

![the-template-file](https://user-images.githubusercontent.com/4063887/117486425-3e640c80-af2f-11eb-99c8-f0081c65f597.png)

If we used `genesis_custom_block_edit_block`, it would mean editors [wouldn't see it](https://github.com/studiopress/genesis-custom-blocks/blob/1ba74a3f25908f49b28b32b26a26396a747b7153/tests/php/Integration/TestPostCapabilities.php#L85). 

But either way would probably be fine.